### DISCUSSION TITLE
Updating via ubuntu's apt/apt-get

### DISCUSSION THREAD

**Discussion by phoenixbackups** [kanidm/kanidm#2651] [Q&A]
Hello,
I installed Kanidm via the .deb packages. I used the Kanidm docs to build the .deb packages.  How would I go about updating/upgrading these? Is there a repo I can add?

**Comment by yaleman** [comment]
There's a PPA here for regular use but they don't have the stable releases -  https://github.com/kanidm/kanidm_ppa

If you want stable packages you'll need to build them yourself as we don't have anyone actively maintaining that method.
